src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -2,6 +2,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <OpenApiEnabled Condition="'$(OpenApiEnabled)' == ''">true</OpenApiEnabled>
+    <OpenApiServiceProfile Condition="'$(OpenApiServiceProfile)' == ''">backend</OpenApiServiceProfile>
```

Comment:
Should we do user input validation?

Reply:
good call out, added it and improved tests